packages/hash/api/src/model/knowledge/orgMembership.model.ts

```diff
@@ -96,36 +96,48 @@ export default class extends EntityModel {
    * Get the org linked to the org membership.
    */
   async getOrg(graphApi: GraphApi): Promise<OrgModel> {
-    const outgoingOrgLinks = await LinkModel.getByQuery(graphApi, {
-      all: [
-        {
-          equal: [{ path: ["source", "id"] }, { parameter: this.entityId }],
-        },
-        {
-          equal: [
-            { path: ["type", "versionedUri"] },
-            {
-              parameter: SYSTEM_TYPES.linkType.ofOrg.schema.$id,
-            },
-          ],
-        },
-      ],
-    });
+    const outgoingOrgLinkEntityModels = await LinkEntityModel.getByQuery(
+      graphApi,
+      {
+        all: [
+          {
+            equal: [
+              { path: ["leftEntity", "uuid"] },
+              { parameter: this.entityUuid },
+            ],
+          },
+          {
+            equal: [
+              { path: ["type", "versionedUri"] },
+              {
+                parameter: SYSTEM_TYPES.linkEntityType.ofOrg.schema.$id,
```

Comment:
Do we want to add a todo here to remove this when we just make `orgMembership` a link type? (Which I had assumed we'd do now but probably better to keep it self-contained 👍 )

Reply:
Sure, this is currently [a separate task](https://app.asana.com/0/0/1203371754468058/f) yes. Not sure this is a great place for that todo, as it affects the model class as a whole. Have added on in https://github.com/hashintel/hash/pull/1435/commits/5f81afc94e5addf13a9f357548d36eb118902b53